from Exceptions import exist_in_favorites
from Exceptions import invalid_name
from Exceptions import invalid_number
from Exceptions import name_exist
from Exceptions import number_exist
